.github/workflows/testing.yml

```
@@ -132,7 +133,7 @@ jobs:
           - extra-toggles: prometheus
     steps:
       - name: Reclaim space on GHA host (if the job needs it)
-        if: ${{ matrix.ovn != '' }}
+        if: ${{ matrix.ovn != '' }} || ${{ matrix.ovn-ic != '' }}
```

```suggestion
        if: ${{ matrix.ovn != '' }} && ${{ matrix.ovn-ic != '' }}
```

Shouldn't this be an `and` operation? If we want to avoid both